### PR TITLE
docs: add automaticallyAdjustKeyboardInsets to scrollview props

### PR DESCRIPTION
**Pull request by jeremybarbet**
New props landed [here](https://github.com/facebook/react-native/commit/49a1460a379e3a71358fb38888477ce6ea17e81a)

---

Sync with others repo
- https://github.com/facebook/react-native/pull/33948
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60618

### CODE REVIEW

**Comment by cortinico** [comment]
Don't you mind backporting this to 0.68 as well?

File: docs/scrollview.md [hunk 1]
```diff
@@ -112,6 +112,16 @@ Controls whether iOS should automatically adjust the content inset for scroll vi
 
 ---
 
+### `automaticallyAdjustKeyboardInsets` <div class="label ios">iOS</div>
+
+Controls whether the ScrollView should automatically adjust its contentInset and scrollViewInsets when the Keyboard changes its size.
```
Comment (Simek) [inline docs/scrollview.md:117]:
```suggestion
Controls whether the ScrollView should automatically adjust its `contentInset` and `scrollViewInsets` when the Keyboard changes its size.
```

Let's format the prop names in the description.

File: website/versioned_docs/version-0.68/scrollview.md [hunk 2]
```diff
@@ -112,6 +112,16 @@ Controls whether iOS should automatically adjust the content inset for scroll vi
 
 ---
 
+### `automaticallyAdjustKeyboardInsets` <div class="label ios">iOS</div>
+
+Controls whether the ScrollView should automatically adjust its contentInset and scrollViewInsets when the Keyboard changes its size.
```
Comment (Simek) [inline website/versioned_docs/version-0.68/scrollview.md:117]:
```suggestion
Controls whether the ScrollView should automatically adjust its `contentInset` and `scrollViewInsets` when the Keyboard changes its size.
```

Same as above.

**Review by Simek — APPROVED** [review]
Thank for the updates @jeremybarbet, let's ship it! 👍

**Pull request merged**